cmd/ocitools/validate.go

```diff
@@ -84,7 +84,7 @@ var bundleValidateCommand = cli.Command{
 			return fmt.Errorf("The root path %q is not a directory.", rootfsPath)
 		}
 
-		hostCheck := context.Bool("host-specific")
+		hostCheck := hostSpecific || context.Bool("host-specific")
```

Comment:
On Thu, Jul 21, 2016 at 12:33:31PM -0700, hmeng-19 wrote:
> I use a global var here to avoid multiple func calls to
> `context.GlobalBool`.

What's the drawback to multiple GlobalBool calls?  I like that more
than a package global, since it keeps the “single, unambiguous,
authoritative” representation of that setting [1].

[1]: https://en.wikipedia.org/wiki/Don't_repeat_yourself